pkg/cli/pod/pod_test.go

```diff
@@ -202,7 +203,7 @@ func buildPod(namespace, name string, labels map[string]string, annotations map[
 			Namespace:         namespace,
 			Labels:            labels,
 			Annotations:       annotations,
-			CreationTimestamp: metav1.Now(),
+			CreationTimestamp: metav1.Time{Time: time.Now().Add(-3e11)},
```

Comment:
How about use `time.Now().UTC().AddDate(0, 0, -3)` to just make the time accurate to the day?

Reply:
I modified it and it seems to be solvable. I simulated sleeping for one minute locally and no error occurred.
thanks. Good method!